From e3ad3deb6a6e79284f3748fa7410311d87df91c5 Mon Sep 17 00:00:00 2001
From: Henrique Nakashima <hnakashima@chromium.org>
Date: Tue, 4 Sep 2018 16:49:51 +0000
Subject: [PATCH] IWYU: stdint.h in pdfium_mem_buffer_file_write.h for uint8_t

Bug: 879900
Change-Id: I9c15d1c280a23c53d31f2d72c9d0d1db79eab886
Reviewed-on: https://chromium-review.googlesource.com/1204410
Reviewed-by: Lei Zhang <thestig@chromium.org>
Commit-Queue: Henrique Nakashima <hnakashima@chromium.org>
Cr-Commit-Position: refs/heads/master@{#588547}
---
 pdf/pdfium/pdfium_mem_buffer_file_write.h | 1 +
 1 file changed, 1 insertion(+)

--- a/pdf/pdfium/pdfium_mem_buffer_file_write.h
+++ b/pdf/pdfium/pdfium_mem_buffer_file_write.h
@@ -6,6 +6,7 @@
 #define PDF_PDFIUM_PDFIUM_MEM_BUFFER_FILE_WRITE_H_
 
 #include <stddef.h>
+#include <stdint.h>
 
 #include <vector>
 
